operators/t8c/42.8.0/t8c-operator.v42.8.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,201 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[{"apiVersion":"charts.helm.k8s.io/v1","kind":"Xl","metadata":{"name":"xl-release"},"spec":{"global":{"tag":"8.5.0"}}}]'
+    capabilities: Basic Install
+    categories: Monitoring
+    certified: "false"
+    containerImage: turbonomic/t8c-operator:42.8
```

Comment:
Please specify registry, because OCP pipeline is defaulting to wrong one in this case.

Reply:
thanks, updated